public/app/plugins/datasource/prometheus/querybuilder/components/MetricEncyclopediaModal.tsx

```diff
@@ -129,6 +129,7 @@ export const MetricEncyclopediaModal = (props: Props) => {
   const [filteredMetricCount, setFilteredMetricCount] = useState<number>();
   // backend search metric names by text
   const [useBackend, setUseBackend] = useState<boolean>(false);
+  const [disableTextWrap, setDisableTextWrap] = useState<boolean>(false);
```

Comment:
I feel like we may remove this at some point but for now I am liking the feeling of seeing the difference between lines of equal size (no text wrap) and lines with greater height. Nice!